client/app/components/sections/SearchPage/SearchPage.js

```diff
@@ -8,7 +8,7 @@ class SearchPage extends Component {
 
   constructor(props, context) {
     super(props, context);
-    this.listings = [];
+    this.listings = props.searchPage.listings || [];
```

Comment:
This should be currentListings in the future. (Listings filtered by currentPage indexes)